src/libmongoc/tests/test-mongoc-sdam.c

```diff
@@ -86,6 +96,41 @@ _topology_has_description (mongoc_topology_description_t *topology,
                       BSON_ITER_HOLDS_INT32 (&iter));
          expected_generation = bson_iter_int32 (&iter);
          ASSERT_CMPINT32 (expected_generation, ==, sd->generation);
+      } else if (strcmp ("logicalSessionTimeoutMinutes",
```

Comment:
These changes are to support the new [discover_load_balancer.yml](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/tests/load-balanced/discover_load_balancer.yml) test. The test asserts many fields on a load balancer are unset by setting the expectation to `null`.